resources/migrations/000_migrations.yaml

```diff
@@ -15337,6 +15337,32 @@ databaseChangeLog:
                     nullable: false
                   defaultValue: '#31698A'
 
+  - changeSet:
+      id: v48.00-021
+      author: piranha
+      comment: 'Cards store Metabase version used to create them'
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: metabase_version
+                  type: varchar(100)
+                  remarks: 'Metabase version which made a given revision.'
```

Comment:
Should the remarks be adjusted to say "Metabase version used to create the card" for clarity?